source/manual/github-unavailable.html.md

```diff
@@ -18,8 +18,8 @@ build and deploy a container image from your workstation.
 Run the following commands from the root directory of the repository.
 
 1. [Set up CodeCommit on your machine](/manual/howto-checkout-and-commit-to-codecommit.html#install-dependencies-and-set-up-local-environment) if you haven't already.
-1. Follow the [CodeCommit guide](/manual/howto-checkout-and-commit-to-codecommit.html#quick-reference-guide) to clone the repo and commit/push changes if needed.
-1. Set the image tag, image registry and image repository names that you will use by running the following commands. You don't need to modify any of the values in the commands.
+2. Follow the [CodeCommit guide](/manual/howto-checkout-and-commit-to-codecommit.html#quick-reference-guide) to clone the repo and commit/push changes if needed.
```

Comment:
probs overkill but we could potentially mention that if it's a repo you've got in the `/govuk` dir you need to clone it somewhere else. Although i'm sure people will be able to figure that out

Reply:
I've added a comment about this within the Clone from CodeCommit guide.